src/components/executive-education-2u/ExecutiveEducation2UPage.jsx

```diff
@@ -45,30 +50,44 @@ const ExecutiveEducation2UPage = () => {
     }
   }, [activeQueryParams, enterpriseConfig]);
 
-  if (!isExecEd2UFulfillmentEnabled) {
-    return (
-      <NotFoundPage />
-    );
-  }
-
-  const isLoading = isLoadingContentMetadata;
-  const { name: enterpriseName } = enterpriseConfig;
-
   const pageTitle = 'Share course enrollment information';
   const queryParams = {
     failureReason: activeQueryParams.get('failure_reason'),
     httpReferrer: activeQueryParams.get('http_referer'),
     sku: activeQueryParams.get('sku'),
   };
 
-  const handleCheckoutSuccess = (response) => {
-    if (!response?.receiptPageUrl) {
-      logError('Unable to redirect to receipt page due to missing receipt page URL.');
-      return;
+  const courseMetadata = useMemo(() => {
+    if (contentMetadata) {
+      const activeCourseRun = getActiveCourseRun(contentMetadata);
+      const organizationDetails = getCourseOrganizationDetails(contentMetadata);
+      return {
+        organizationImage: organizationDetails.organizationLogo,
+        organizationName: organizationDetails.organizationName,
+        title: contentMetadata.title,
+        startDate: moment(activeCourseRun?.start).format(DATE_FORMAT),
+        duration: `${activeCourseRun?.weeksToComplete} Week${activeCourseRun?.weeksToComplete >= 1 ? '' : 's'}`,
+        priceDetails: getExecutiveEducationCoursePrice(contentMetadata),
+      };
     }
-    global.location.assign(response.receiptPageUrl);
+    return {};
+  }, [contentMetadata]);
+
+  const handleCheckoutSuccess = () => {
+    push({
+      pathname: `/${enterpriseConfig.slug}/executive-education-2u/enrollment-completed`,
+      state: {
+        data: courseMetadata,
+      },
+    });
   };
 
+  if (!isExecEd2UFulfillmentEnabled) {
+    return (
+      <NotFoundPage />
+    );
+  }
+
   return (
     <Container size="lg" className="py-5">
```

Comment:
The mocks depict a background color (i.e., `.bg-light-200`) for the PLI enrollment flow. Was this intentionally left out or should it be included? If it should be included, it could be implemented by wrapping this `Container` with:

```jsx
<div className="bg-light-200">
  <Container>{...}</Container>
</div>
```

<img width="1480" alt="image" src="https://user-images.githubusercontent.com/2828721/231522887-d8b7764a-0eab-4c37-91ed-98efe50139be.png">